### PR TITLE
BREAKING CHANGE: Migrate to Jest and Webpack to be more compatible with Lit 3 

### PR DESCRIPTION
**Pull request by ataromoku**
## Related jira:
https://desire2learn.atlassian.net/browse/PHNX-1954

## This PR:
* Replaces `Karma, Browserify, Babelify` with `Jest, Webpack`. 
* Bumps `bump-react-valence-ui-iframe`.
* Only affects the tests and demo, it doesn't modify the logic itself.

---

## Why do we need this?
While configuring the `Babelify` for Lit 3, we had to update some dependencies that bring along others. As a result, we arrive at a configuration where we can no longer stably work with `React` versions earlier than `16`. 
### Why Jest? 
Jest is perfectly compatible with React. It debugs well and doesn't require a bundler to execute tests.

### CODE REVIEW

**Review by brissons — COMMENTED** [review]
a couple of nits but otherwise lgtm!

File: .eslintrc [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "parserOptions": {
     "ecmaFeatures": {
       "jsx": true
-    }
+    },
+	"ecmaVersion": 2018
```
[the remark this answers is not in the document]
Comment (ataromoku) [reply .eslintrc:7]: Unfortunately, the last version this code works with is the 2018

File: src/plugins/generic/__tests__/viewerTests.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-var React = require('react'), //eslint-disable-line no-unused-vars
-	TestUtils = require( 'react-addons-test-utils' ),
+var React = require('react'), //eslint-disable-line no-unused-
```
[the remark this answers is not in the document]
Comment (ataromoku) [reply src/plugins/generic/__tests__/viewerTests.js]: Removed 👍

**Review by brissons — APPROVED** [review]

File: src/plugins/pdf/native/__tests__/nativePluginTests.js
[the remark this answers is not in the document]
Comment (ataromoku) [reply src/plugins/pdf/native/__tests__/nativePluginTests.js:1]: It's weird, but test fails with `ReferenceError: native is not defined` if we name this variable differently from exported here https://github.com/Brightspace/react-valence-ui-fileviewer/blob/1af544d860a1bb3e09724ecb48de2553903e1c52/src/plugins/pdf/native/native.js#L20

File: src/plugins/image/__tests__/imagePluginTests.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ describe('Image Plugin', function() {
 	});
 
 	it('should return a view', function() {
-		var viewer = ImagePlugin.getComponent();
+		var viewer = ImagePlugin.getComponent({src: 'test'});
```
[the remark this answers is not in the document]
Comment (ataromoku) [reply src/plugins/image/__tests__/imagePluginTests.js:21]: Thx, any value will do here, so I removed the .html for consistency

File: src/plugins/generic/__tests__/viewerTests.js [hunk 4]
```diff
@@ -41,6 +41,7 @@ describe('Generic Viewer', function() {
 
 	it('should render wrapper with expected class name', function() {
 		var elem = TestUtils.renderIntoDocument(
+
```
[the remark this answers is not in the document]
Comment (ataromoku) [reply src/plugins/generic/__tests__/viewerTests.js]: Unintentionally 💯

**Review by prajapac — APPROVED** [review]

**Pull request merged**